CONTRIBUTING.md

```diff
@@ -344,9 +344,59 @@ Details for each point and good commit message examples can be found on https://
 
 ## Committers
 
-Presto committers are defined as [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) and documented in the project's [`CODEOWNERS`](CODEOWNERS) file.  Each line in the `CODEOWNERS` file defines a module or submodule that the committer has the rights to approve.  New modules and submodules for CODEOWNERS may be added as needed.
+Presto has two levels of committers: module committers, and project committers.  Presto committers are defined as [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) and documented in the project's [`CODEOWNERS`](CODEOWNERS) file, and are either directly referenced in the file, or through a Github team. Each line in the `CODEOWNERS` file defines a module or submodule that the committer or team has the rights to approve.  New modules and submodules for [`CODEOWNERS`](CODEOWNERS)  may be added as needed.
 
-New committers are approved by majority vote of the TSC ([see TSC charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md)).  To become a committer, reach out to an [existing TSC member](https://github.com/prestodb/tsc#members) and ask for their feedback on your eligibility (see: [How to become a Presto Committer?](https://github.com/prestodb/presto/wiki/How-to-become-a-Presto-committer%3F)).  Note: to expedite the process, consider creating a document that outlines your Github stats, such as the number of reviews, lines of code added, number of PRs, and outlines particularly outstanding code and review contributions.  If the TSC member believes you are eligible, they will submit your nomination to a vote by the TSC, typically in the form of a PR that adds your handle to the `CODEOWNERS` file.  The process is complete once the PR is merged.
+### Baseline expectations from committers
+
+Both module and project committers must demonstrate technical mastery and commitment to the project’s community and growth.  In particular, they must show evidence that they are aligned with the goals outlined in [ARCHITECTURE](ARCHITECTURE.md), and demonstrate kindness and professionalism; help others in the project; and work collaboratively as a team.
```

Comment:
in [ARCHITECTURE](ARCHITECTURE.md), demonstrate kindness and professionalism, help others in the project, and work collaboratively as a team.

(no semicolons)